### PR TITLE
The ”IPC“communication protocol is changed to TCP， geometricError adjust scale

### PR DESCRIPTION
**Pull request by jiawenquan**
Since the `ipc `communication protocol cannot run on Windows, I changed `ipc `to `tcp`.
`geometricError`  I found a suitable scale

### CODE REVIEW

File: py3dtiles/convert.py [hunk 1]
```diff
@@ -411,7 +411,7 @@ def add_tasks_to_process(state, name, task, point_count):
     context = zmq.Context()
 
     zmq_skt = context.socket(zmq.ROUTER)
-    zmq_skt.bind('ipc:///tmp/py3dtiles1')
+    zmq_skt.bind('tcp://127.0.0.1:1234')
```
Comment (peppsac) [inline py3dtiles/convert.py:413]: To avoid dealing with port number, I'd suggest the following:
```suggestion
    zmq_skt.bind('tcp://127.0.0.1:0')
    zmq_tcp_address = zmq_skt.getsockopt(zmq.LAST_ENDPOINT)
```
And then pass this `zmq_tcp_address` as the 1st argument to zmq_process

File: py3dtiles/points/node.py [hunk 2]
```diff
@@ -208,7 +208,7 @@ def to_tileset(executor, name, parent_aabb, parent_spacing, folder, scale):
                     0, 0, half_size[2]]
             },
             # geometricError is in meter so cancel scale
-            'geometricError': 10 * node.spacing / scale[0],
+            'geometricError': 20 * node.spacing / scale[0],
```
Comment (peppsac) [inline py3dtiles/points/node.py]: Let's not mix unrelated commits in a single PR.

I'm fine with changing the `geometricError` value, but you'll need to show that your proposed value works better on a couple of models.

**Review by peppsac — COMMENTED** [review]
Thanks for PR !

I added a minor suggestion, and then it should be mergeable.

File: py3dtiles/points/node.py (hunk 2 above)
Comment (jiawenquan) [reply py3dtiles/points/node.py]: Sorry, this is not a good commit, 
I think we can provide a parameter for easy modification

File: py3dtiles/convert.py (hunk 1 above)
Comment (jiawenquan) [reply py3dtiles/convert.py:413]: That's great

Comment (chenaza) [reply py3dtiles/convert.py]: Can I ask how to pass this zmq_tcp_address as the 1st argument to zmq_process?

**Comment by delhomer** [comment]
Moved to https://gitlab.com/Oslandia/py3dtiles/-/merge_requests/39
